src/scss/qld-variables.scss

```diff
@@ -31,16 +34,16 @@ $cobrand-orange : #FD855D;
 
 
 //QGOV Brand Mapping - QLD.GOV.AU
-$qld-brand-primary-dark : #121212; //$qld-dark-blue; 
+$qld-brand-primary-dark : #121212; //$qld-dark-blue;
```

Comment:
We should assign 121212 to a named colour variable in a future commit

Reply:
Agree. Not too sure where the `#121212` came from or if it's the right colour? Maybe a leftover from some early dark theme testing? 

Should it be changed back to the commented `$qld-dark-blue`?